src/js/options.js

```diff
@@ -53,11 +53,20 @@ async function enableDisableReplaceTab() {
   await browser.storage.local.set({replaceTabEnabled: !!checkbox.checked});
 }
 
+async function changeTheme() {
+  const theme = document.querySelector("#changeTheme");
+  await browser.storage.local.set({currentTheme: theme.value});
+  await browser.storage.local.set({currentThemeId: theme.selectedIndex});
+}
```

Comment:
It looks like this function is only called via `document.querySelector("#changeTheme").addEventListener(...)`, so, instead of calling `document.querySelector("#changeTheme")` again, we can reference the element via either `this` or `currentTarget`?

```suggestion
async function changeTheme(event) {
  const theme = event.currentTarget;
  await browser.storage.local.set({currentTheme: theme.value});
  await browser.storage.local.set({currentThemeId: theme.selectedIndex});
}
```